src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -1203,4 +1205,29 @@
                         StablePackages="@(StablePackage)"
                         ModuleToPackages="@(ModuleToPackage)" />
   </Target>
+
+  <Target Name="CreateLayout"
```

Comment:
It'll work either way.   That's why I made it take an array.  I just put it here as a shortcut.  I'm fine with this target not being in packaging.targets and only being in your layout project.